aiida/cmdline/params/options/verbosity.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Option for setting global verbosity of verdi command line interface."""
+
+import click
+from aiida.common.log import VERDI_LOGGER, LOG_LEVELS
+from aiida.manage.configuration import get_config_option
+from .overridable import OverridableOption
+
+
+class VerbosityOption(OverridableOption):
+    """
+    Option for controlling verbosity of loggers (default: "verdi" logger).
+
+    Example::
+
+        VERBOSITY = VerbosityOption('-v', '--verbosity', logger=MY_LOGGER)
+
+        @click.command()
+        @VERBOSITY()
+        def computer_setup(computer_name):
+            click.echo("Setting up computer {}".format(computername))
+
+        computer_setup -v DEBUG
+
+    """
+
+    def __init__(self, *args, **kwargs):
+        """
+        Store the default args and kwargs.
+
+        :param args: default arguments to be used for the option
+        :param kwargs: default keyword arguments to be used that can be overridden in the call
+        """
+        levels_str = ', '.join(LOG_LEVELS.keys())
+        self.logger = kwargs.pop('logger', VERDI_LOGGER)
+
+        kwargs.setdefault('default', get_config_option('logging.verdi_loglevel'))
+        kwargs.setdefault('show_default', True)
+        kwargs.setdefault('expose_value', False)
```

Comment:
You should till expose the value, to allow this for example to be used with progress bars or any other potential means of reporting that are not loggers